openhands/server/config/openhands_config.py

```diff
@@ -1,22 +1,40 @@
 import os
 
+from fastapi import HTTPException
+
 from openhands.server.types import AppMode, OpenhandsConfigInterface
 from openhands.utils.import_utils import import_from
 
 
 class OpenhandsConfig(OpenhandsConfigInterface):
-    CONFIG_PATH = os.environ.get('OPENHANDS_CONFIG_PATH', None)
-    APP_MODE = AppMode.OSS
-    POSTHOG_CLIENT_KEY = 'phc_3ESMmY9SgqEAGBB6sMGK5ayYHkeUuknH2vP6FmWH9RA'
-    GITHUB_CLIENT_ID = os.environ.get('GITHUB_APP_CLIENT_ID', '')
-    ATTACH_SESSION_MIDDLEWARE_PATH = (
+    config_path = os.environ.get('OPENHANDS_CONFIG_PATH', None)
+    app_mode = AppMode.OSS
+    posthog_client_key = 'phc_3ESMmY9SgqEAGBB6sMGK5ayYHkeUuknH2vP6FmWH9RA'
+    github_client_id = os.environ.get('GITHUB_APP_CLIENT_ID', '')
+    attach_session_middleware_path = (
         'openhands.server.middleware.AttachSessionMiddleware'
     )
 
     def verify_config(self):
-        if self.CONFIG_PATH:
+        if self.config_path:
             raise ValueError('Unexpected config path provided')
 
+    def verify_github_repo_list(self, installation_id):
+        if self.app_mode == AppMode.OSS and installation_id:
+            raise HTTPException(
+                status_code=400,
+                detail='Unexpected installation ID',
```

Comment:
This is verifying if there's an unexpected `installation_id`, it's no longer verifying if there's a missing `installation_id`. Is that intended?

Reply:
Yeah, this config is for OH OSS (it should be using user's PAT), so no `installation_id` should be present

OH config for saas gets dynamically imported [here](https://github.com/All-Hands-AI/OpenHands/pull/5371/files#diff-136d34f31c20675994b2c87979c4245a79a9343bafbe8643aca0ad50193bc701R48), which will be responsible for checking if `installation_id` is missing

Reply:
Oh of course, okay!